internal/command/jsonconfig/config.go

```diff
@@ -279,6 +279,7 @@ func marshalProviderConfigs(
 		for _, ppc := range mc.Providers {
 			// These provider names include aliases, if set
 			moduleProviderName := ppc.InChild.String()
+			// TODO/Oleksandr: update this field to resolved provider after graph execution
```

Comment:
I think long term we need to change the jsonconfig format here.  The only major impact to this is the plan format , which is not supported between tofu versions anyways.